bin/xud

```diff
@@ -4,16 +4,16 @@ const Xud = require('../dist/Xud').default;
 
 const { argv } = require('yargs')
   .options({
+    dbPath: {
+      describe: 'The file path for the database, or ":memory:" if the database should be kept in memory',
```

Comment:
Do we really need `or ":memory:" if the database should be kept in memory`? A normal user or exchange would not even consider using this and if you think it is beneficial to developers you should write it into either the contribution guidelines or some other document.

Reply:
Agreed.